### ISSUE TITLE
Some kind of support for directives

### ISSUE THREAD

**Issue by vjekob** [vjekob/al-objid#12]
Right now, Ninja will see this as two objects:

```al
#if something
codeunit 50001....
#else
codeunit 7013010..
#endif
```

Also, object ID assignment will occur from the first available range.

Some kind of support for this is needed.

Any ideas how this should behave, functionally?

**Comment by waldo1001** [comment]
Both ID's are consumed, are they not?

**Comment by vjekob** [comment]
This issue is resolved in AL Object ID Ninja v2.3.1 and @vjeko.com/al-parser-ninja v1.0.8
